provisioning/transport/mqtt/src/mqtt.ts

```diff
@@ -72,7 +74,11 @@ export class Mqtt extends EventEmitter implements X509ProvisioningTransport, Sym
             /* Codes_SRS_NODE_PROVISIONING_MQTT_18_029: [ When `queryOperationStatus` receives an error from the service, it shall call `callback` passing in the error.] */
             handler(translateError('incoming message failure', status, payloadJson, { topic: topic, payload: payloadJson }));
           }
+        } else {
+          debug('received an unanticipated response: ' + topic);
```

Comment:
> unanticipated response [](start = 28, length = 23)

nit: "unknown request id - then print the rid. (and then maybe the whole topic just in case we'd have this because of a parsing issue)

Reply:
Done.

---
In reply to: [280187470](https://github.com/Azure/azure-iot-sdk-node/pull/549#discussion_r280187470) [](ancestors = 280187470)